source/beamline/AMPVControl.cpp

```diff
@@ -459,8 +483,8 @@ void AMPVwStatusControl::onReadPVInitialized()
 }
 
 void AMPVwStatusControl::onWritePVInitialized() {
-	lowLimitValue_ = writePV_->lowerControlLimit();
-	highLimitValue_ = writePV_->upperControlLimit();
```

Comment:
This change will make AMPVwStatusAndUnitConversionControl not work because it only emits the signal in setLow/HighLimitValue and doesn't set the value..